<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>653</width>
    <height>424</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>350</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #D1F2EB;
    border-radius: 12px;
    font-weight: bold;
    color: #145A32;
    border: 2px solid transparent;
}

QPushButton:hover {
    background-color: #A9DFBF;  /* 鼠标移上去变深 */
    border: 2px solid #58D68D;  /* 出现边框线 */
}

QPushButton:pressed {
    background-color: #7DCEA0;  /* 点击时更深 */
}</string>
   </property>
   <property name="text">
    <string>对话</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>50</y>
     <width>141</width>
     <height>191</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid rgba(255, 255, 255, 40);  /* 极细的白边，增加高级感 */
border-radius: 10px;                       /* 大圆角 */</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_2">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>260</y>
     <width>141</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid rgba(255, 255, 255, 40);  /* 极细的白边，增加高级感 */
border-radius: 11px;                       /* 大圆角 */</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>350</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #D1F2EB;
    border-radius: 12px;
    font-weight: bold;
    color: #145A32;
    border: 2px solid transparent;
}

QPushButton:hover {
    background-color: #A9DFBF;  /* 鼠标移上去变深 */
    border: 2px solid #58D68D;  /* 出现边框线 */
}

QPushButton:pressed {
    background-color: #7DCEA0;  /* 点击时更深 */
}</string>
   </property>
   <property name="text">
    <string>发送</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>651</width>
     <height>421</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="te_recv">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>411</width>
     <height>171</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid rgba(255, 255, 255, 40);  /* 极细的白边，增加高级感 */
border-radius: 10px;                       /* 大圆角 */</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_IP">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>390</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="te_send">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>250</y>
     <width>411</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid rgba(255, 255, 255, 40);  /* 极细的白边，增加高级感 */
border-radius: 10px;                       /* 大圆角 */</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_PORT">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>390</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>端口</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_IP">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>390</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>IP</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_tip1">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>230</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>请输入信息</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_tip2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>聊天信息</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_send">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>290</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>发送</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_Port">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>390</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_connect">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>390</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>连接</string>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>pushButton</zorder>
  <zorder>textEdit</zorder>
  <zorder>textEdit_2</zorder>
  <zorder>pushButton_2</zorder>
  <zorder>te_recv</zorder>
  <zorder>le_IP</zorder>
  <zorder>te_send</zorder>
  <zorder>label_PORT</zorder>
  <zorder>label_IP</zorder>
  <zorder>label_tip1</zorder>
  <zorder>label_tip2</zorder>
  <zorder>pb_send</zorder>
  <zorder>le_Port</zorder>
  <zorder>pb_connect</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
